Smarty/templates/modules/Calendar4You/CalendarView.tpl

```diff
@@ -456,6 +456,23 @@ function hideITSEventInfo(){
     jQuery('#event_info_content').html('');
 
 }
+
```

Comment:
I like this idea but not exactly how it is implemented, I don't like the idea of having the toolbar hovering over the calendar. I would prefer for it to disappear and have the calendar adapt to the available space. Based on this code, I gave it a try and almost got it working but it breaks on changes from month/week/day.
I'll have another go at it when I can.